app/views/projects/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= @project.name %></h1>
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%= @project.contestants.count %></p>
+<% experience_array = @project.contestants.map do |contestant| %>
+    <% contestant.years_of_experience %>
+    <% end %>
+<p>Average Contestant Experience: <%= (experience_array.sum).to_f / (experience_array.count) %> years</p>
```

Comment:
See, this `count` is ok, because it's not AR. 😉 The only issue with your implementation here is that we are using Ruby to process data that we get from AR. (i.e. the call to `@project.contestants`). What we want is for this calculation of the average years of exp to be handled on the db. Therefore, we want to create a model method to do that.

```ruby
class Contestant < ApplicationModel
  # [...]

  def self.average_years_of_exp
    average(:years_of_experience)
  end
end
```

This could have also been placed as an instance method on the `Project` model, like so:

```ruby
class Project < ApplicationModel
  # [...]

  def contestant_average_years_of_exp
    contestants.average(:years_of_experience)
  end
end
```

Choose whichever one makes the most sense to you.